homeassistant/components/nibe_heatpump/config_flow.py

```diff
@@ -31,9 +31,9 @@
     {
         vol.Required(CONF_MODEL): vol.In([e.name for e in Model]),
         vol.Required(CONF_IP_ADDRESS): str,
-        vol.Required(CONF_LISTENING_PORT): cv.port,
-        vol.Required(CONF_REMOTE_READ_PORT): cv.port,
-        vol.Required(CONF_REMOTE_WRITE_PORT): cv.port,
+        vol.Required(CONF_LISTENING_PORT, default=9999): cv.port,
```

Comment:
Need to be optional to work with default.

Reply:
Hmm, really? Wasn't aware of that. Should I change it to a 'suggested' value seeing as these ports really are required?